cra/sample/pr-pipeline.yaml

```diff
@@ -0,0 +1,326 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pr-pipeline
+spec:
+  params:
+    - name: repository
+      description: the git repo
+    - name: branch
+      description: branch
+    - name: target-branch
+      description: target branch
+    - name: api-key
+      description: The IBM Cloud API key is used to access the IBM Cloud Kubernetes
+        Service API and interact with the cluster. You can obtain your API key with
+        'bx iam api-key-create' or via the console at https://cloud.ibm.com/iam#/apikeys
+        by clicking **Create API key** (Each API key only can be viewed once).
+    - name: repository-integration
+      description: the repo integration name
+    - name: commit-id
+      description: commit id
+    - name: commit-timestamp
+      description: commit timestamp
+    - name: pipeline-debug
+      description: toggles debug mode for the pipeline
+    - name: pr-url
+      description: pr url
+    - name: target-commit-id
+      description: target branch commit id
+    - name: project-id
+      description: project id
+      default: ""
+    - name: scm-type
+      description: source code type used (github, github-ent, gitlab)   
+
+  workspaces:
+    - name: artifacts
+
+  tasks:
+    - name: code-fetch-credentials
+      taskRef:
+        name: git-get-credentials
+      workspaces:
+        - name: artifacts
+          workspace: artifacts
+        - name: secrets
+          workspace: artifacts
+      params:
+        - name: repository
+          value: $(params.repository)
+        - name: repository-integration
+          value: $(params.repository-integration)
+        - name: ibmcloud-api-key-secret-key
```

Comment:
this seems that api-key is the "key" that will/should contain the api key value in the CD Tekton Pipeline environment properties ?
Correct ?